### PR TITLE
Added info about tokens to FAQ and a link to them from the funding yo…

### PR DESCRIPTION
**Pull request by walkerlj0**
…page, unable to serve locally and test

### Summary

This pull request makes the following changes:

- [ ] Adds to FAQ ### How does Lilypad use blockchain, and why do I need both ETH and Lilypad tokens to run a job?
- [ ] Puts a link to FAQ in lilypad/lilypad-testnet/quick-start/funding-your-wallet-from-faucet.md

Explain the motivation for making these changes.
It's unclear to users why we are funding a wallet with two tokens, esp if user isn't web3 native

### Task/Issue reference

Closes: add_link_here

### Test plan

Tried to serve locally with Gitbook, but I cannot, for some reason it's installing the wrong gitbook version. If you have any advice on how to setup and serve locally let me know

### Details (optional)
It's up to you if you think this is relevant to include! It's a suggestion. Another option would be to add a brief explanation in the 

### Related issues or PRs (optional)

Add any related issues or PRs.

### CODE REVIEW

File: lilypad/faqs.md [hunk 1]
```diff
@@ -57,7 +57,18 @@ Phases 2 and onward will provide rewards for Lilypad modules created as well as
 
 ### How do I check my Lilybit\_ rewards?
 
-You can check your rewards by pasting your nodes wallet address into the following interfaces:
+You can check your rewards by pasting your nodes wallet address into the following inteerfaces:
+
+### How does Lilypad use blockchain, and why do I need both ETH and Lilypad tokens to run a job?
+
+On the Lilypad network, The blockchain is used for
```
Comment (bgins) [inline lilypad/faqs.md:64]:
```suggestion
On the Lilypad network, the blockchain is used for:
```

File: lilypad/faqs.md [hunk 2]
```diff
@@ -57,7 +57,18 @@ Phases 2 and onward will provide rewards for Lilypad modules created as well as
...
+* payment rails
+* storing the deals transparently (on-chain guarantees about the compute)
```
Comment (bgins) [inline lilypad/faqs.md:66]:
```suggestion
* storing deals transparently (on-chain guarantees about the compute)
```

File: lilypad/faqs.md [hunk 3]
```diff
@@ -57,7 +57,18 @@ Phases 2 and onward will provide rewards for Lilypad modules created as well as
...
+* storing any disputes & results
+
+Lilypad Tokens  are used to transact on the Lilypad network. They are used as payment by those who want to run jobs (to the resource providers who run them), and as collateral by resource providers
```
Comment (bgins) [inline lilypad/faqs.md:69]:
```suggestion
Lilypad Tokens are used to transact on the Lilypad network. They are used as payment by those who want to run jobs (to the resource providers who run them), and as collateral by resource providers.
```

File: lilypad/faqs.md [hunk 4]
```diff
@@ -57,7 +57,18 @@ Phases 2 and onward will provide rewards for Lilypad modules created as well as
...
+
+You need ETH tokens to pay for the gas fees for smart contracts the facilitate transactions, and for records of transactions and disputes that are posted to the Ethereum blockchain.
```
Comment (bgins) [inline lilypad/faqs.md:71]:
```suggestion
You need ETH tokens to pay for the gas fees for smart contracts that facilitate transactions, and for records of transactions and disputes that are posted to the Ethereum blockchain.
```

File: lilypad/faqs.md [hunk 5]
```diff
@@ -57,7 +57,18 @@ Phases 2 and onward will provide rewards for Lilypad modules created as well as
...
 
-You can check your rewards by pasting your nodes wallet address into the following interfaces:
+You can check your rewards by pasting your nodes wallet address into the following inteerfaces:
+
+### How does Lilypad use blockchain, and why do I need both ETH and Lilypad tokens to run a job?
+
+On the Lilypad network, The blockchain is used for
+* payment rails
+* storing the deals transparently (on-chain guarantees about the compute)
+* storing any disputes & results
```
Comment (bgins) [inline lilypad/faqs.md:67]:
```suggestion
* storing any disputes and results
```
We may want to clarify "results" here. We store metadata about a completed job on chain, but not the actual job outputs. Maybe "results metadata"? Not quite sure, and maybe I'm overthinking it.

**Review by bgins — APPROVED** [review]
Looks good! Left a few copy edit suggestions and one detail we may want to clarify.

**Review by galaxyxtwo — APPROVED** [review]
good addition! thanks for reviewing @bgins

**Pull request merged**